src/main/resources/thirdparty/ios-deploy/lldbpatch.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/scripts/lldb.py b/src/scripts/lldb.py
+index 89feffb..9b3382a 100644
+--- a/src/scripts/lldb.py
++++ b/src/scripts/lldb.py
+@@ -103,7 +103,7 @@ def autoexit_command(debugger, command, result, internal_dict):
+     printBacktraceTime = time.time() + detectDeadlockTimeout if detectDeadlockTimeout > 0 else None
```

Comment:
is this valid syntax?

Reply:
It is a Python script.
I'm only commenting out the `debugger.GetListener()...` line